<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="backgrounnd">
   <property name="geometry">
    <rect>
     <x>-7</x>
     <y>0</y>
     <width>407</width>
     <height>550</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>GUI.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="light">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>29</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="exit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>68</y>
     <width>70</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="minimize">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="porttext">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>140</y>
     <width>125</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Gulim';&quot;&gt;0~65535&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reset">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>250</y>
     <width>125</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkbox_syn">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>401</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>20</height>
    </size>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkbox_icmp">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>430</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkbox_udp">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>460</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkbox_http">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>490</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkbox_ntp">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>400</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkbox_ssdp">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>430</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkbox_tcp">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>460</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkbox_dns">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>490</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="icon_syn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>__ON.png</pixmap>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkbox_alarm">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>320</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkbox_defender">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>350</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="icon_icmp">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>__ON.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="icon_udp">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>__ON.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="icon_http">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>__ON.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="icon_ntp">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>400</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>__ON.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="icon_ssdp">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>430</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>__ON.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="icon_tcp">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>460</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>__ON.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="icon_dns">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>490</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>__ON.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="icon_alarm">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>__ON.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="icon_defender">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>__OFF.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="button_alarm">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(255, 255, 255, 0);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_defender">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(255, 255, 255, 0);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_syn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(255, 255, 255, 0);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_icmp">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(255, 255, 255, 0);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_udp">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(255, 255, 255, 0);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_http">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(255, 255, 255, 0);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_ntp">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>400</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(255, 255, 255, 0);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_ssdp">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>430</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(255, 255, 255, 0);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_tcp">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>460</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(255, 255, 255, 0);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_dns">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>490</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(255, 255, 255, 0);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="console">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>69</y>
     <width>250</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 63 8pt;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:8pt; font-weight:56; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Segoe UI Semibold'; font-size:14pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="status_console">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>300</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 63 10pt;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:10pt; font-weight:56; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Gulim';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="traffictext">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>191</y>
     <width>125</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Gulim';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="apply">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>285</y>
     <width>125</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="traffic_limit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="traffic_box_1">
   <property name="geometry">
    <rect>
     <x>338</x>
     <y>501</y>
     <width>45</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="traffic_box_2">
   <property name="geometry">
    <rect>
     <x>338</x>
     <y>471</y>
     <width>45</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="traffic_box_3">
   <property name="geometry">
    <rect>
     <x>338</x>
     <y>441</y>
     <width>45</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="traffic_box_4">
   <property name="geometry">
    <rect>
     <x>338</x>
     <y>411</y>
     <width>45</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="traffic_box_5">
   <property name="geometry">
    <rect>
     <x>338</x>
     <y>381</y>
     <width>45</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>traffic_limit</zorder>
  <zorder>backgrounnd</zorder>
  <zorder>light</zorder>
  <zorder>exit</zorder>
  <zorder>minimize</zorder>
  <zorder>porttext</zorder>
  <zorder>reset</zorder>
  <zorder>checkbox_syn</zorder>
  <zorder>checkbox_icmp</zorder>
  <zorder>checkbox_udp</zorder>
  <zorder>checkbox_http</zorder>
  <zorder>checkbox_ntp</zorder>
  <zorder>checkbox_ssdp</zorder>
  <zorder>checkbox_tcp</zorder>
  <zorder>checkbox_dns</zorder>
  <zorder>icon_syn</zorder>
  <zorder>checkbox_alarm</zorder>
  <zorder>checkbox_defender</zorder>
  <zorder>icon_icmp</zorder>
  <zorder>icon_udp</zorder>
  <zorder>icon_http</zorder>
  <zorder>icon_ntp</zorder>
  <zorder>icon_ssdp</zorder>
  <zorder>icon_tcp</zorder>
  <zorder>icon_dns</zorder>
  <zorder>icon_alarm</zorder>
  <zorder>icon_defender</zorder>
  <zorder>button_alarm</zorder>
  <zorder>button_defender</zorder>
  <zorder>button_syn</zorder>
  <zorder>button_icmp</zorder>
  <zorder>button_udp</zorder>
  <zorder>button_http</zorder>
  <zorder>button_ntp</zorder>
  <zorder>button_ssdp</zorder>
  <zorder>button_tcp</zorder>
  <zorder>button_dns</zorder>
  <zorder>console</zorder>
  <zorder>status_console</zorder>
  <zorder>traffictext</zorder>
  <zorder>apply</zorder>
  <zorder>traffic_box_1</zorder>
  <zorder>traffic_box_2</zorder>
  <zorder>traffic_box_3</zorder>
  <zorder>traffic_box_4</zorder>
  <zorder>traffic_box_5</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
